ci/codegen.sh

```diff
@@ -16,7 +16,16 @@ markEndOfBlock "Copying Packages Schema"
 
 markStartOfBlock "Generating code"
 
+if isWindows;
+  then SCHEMA_COMPILER="tools/schema_compiler/win/schema_compiler.exe"
+elif isMacOS;
+  then SCHEMA_COMPILER="tools/schema_compiler/macos/schema_compiler"
+elif isLinux;
```

Comment:
No need for the `isLinux` branch, we don't support it or test on it.

Reply:
Created new PR for these fixes sorry!

https://github.com/spatialos/UnityGDK/pull/113